app/controllers/internal/events_controller.rb

```diff
@@ -1,11 +1,12 @@
 module Internal
   class EventsController < ApplicationController
     layout "internal"
+    include ApplicationHelper
 
     def index
       @event = Event.new(
-        location_name: "dev.to/live",
-        location_url: "https://dev.to",
+        location_name: "#{ApplicationConfig['APP_DOMAIN']}/live",
```

Comment:
Could be `URL.domain` ?

Reply:
Thanks Anna, going to update this now. In retrospect, theres a few other places in the codebase I'm still using `ApplicationConfig['APP_DOMAIN']`, I'll update those in subsequent PR's as I go along.